client-js/app/elements/labrad-registry.html

```diff
@@ -277,26 +289,38 @@
       <h1>Create New Key</h1>
       <paper-input id="newKeyInput" label="Key" autofocus></paper-input>
       <paper-textarea id="newValueInput" label="Value"></paper-textarea>
+      <div class="dialogError">
+        <template is="dom-if" if="{{errorTitle}}"><h4>{{errorTitle}}</h4></template>
+        <p>{{errorMessage}}</p>
```

Comment:
No, the errorTitle is optional. It's a question if its worth wrapping the entire thing in a "if errorMessage", but this is otherwise as intended.